cli/command/client/map.go

```diff
@@ -106,11 +108,25 @@ func ParseSize(size string) (int, error) {
 	return n, nil
 }
 
+func ParseBlockSize(blocksize string) (uint64, error) {
+	blocksize = strings.TrimSuffix(blocksize, "B")
+	m, err := humanize.ParseBytes(blocksize)
```

Comment:
You can look at its [source code](https://github.com/dustin/go-humanize/blob/master/bytes.go)